test/integration/tests/buybacks.ts

```diff
@@ -17,7 +17,7 @@ import { BalancerLBPSwapper, CollateralizationOracle, IVault, IWeightedPool } fr
 import { forceEth } from '../setup/utils';
 const toBN = ethers.BigNumber.from;
 
-describe('e2e-buybacks', function () {
```

Comment:
instead of skipping, you could add a before() hook that checks if the PCVEquityMinter is paused, and if so, unpause it

Reply:
good idea